contracts/SLORegistry.sol

```diff
@@ -17,18 +17,19 @@ contract SLORegistry {
     }
 
     struct SLO {
-        uint256 sloValue;
+        // half storage slot
+        uint120 sloValue;
         SLOType sloType;
     }
-    /**
-     * @dev SLO Registered event
-     * @param sla 1. -
-     * @param sloValue 2. -
-     * @param sloType 3. -
-     */
+
+    /// @dev SLO Registered event
     event SLORegistered(address indexed sla, uint256 sloValue, SLOType sloType);
 
+    /// @notice maximum cap of deviation percent = 25%, base 10000
+    uint16 private constant deviationCapPercent = 2500;
```

Comment:
`deviationCap` better?

We're not using the "percent" suffix for any of the percentage in the protocol.